deploy/template.yaml

```diff
@@ -151,8 +151,8 @@ Mappings:
       OIDCURL: "https://oidc.integration.account.gov.uk/"
       RETURNREDIRECTURL: "https://return.integration.account.gov.uk/callback"
     production:
-      GOVUKNOTIFYTEMPLATEID: "206971e2-d576-4d12-b34f-b0de442d39ab"
-      GOVUKNOTIFYDYNAMICEMAILTEMPLATEID: "f62672d4-ff45-470e-8506-956482061f54"
+      GOVUKNOTIFYTEMPLATEID: "b1f0f510-7501-4e49-b5d4-aced6e36ad90"
```

Comment:
Updating the templateIds for both static and dynamic to fallbackemail template for this temporary fix, will be reverted along with other code changes once the emails are sent for those affected.